queue_services/entity-emailer/src/entity_emailer/worker.py

```diff
@@ -34,7 +34,7 @@
 from entity_queue_common.service import QueueServiceManager
 from entity_queue_common.service_utils import EmailException, QueueException, logger
 from flask import Flask
-from legal_api import db
+from legal_api import db, init_db  # noqa:F401,I001;pylint:disable=unused-import;
```

Comment:
Didn't remove the `db` as in test when mocking, the db attribute is needed for worker. As experimented, removing it caused unit-tests initialization errors. https://github.com/bcgov/lear/blob/feature-db-versioning/queue_services/entity-emailer/tests/conftest.py#L322C1-L322C42

Reply:
What's interesting is that you can remove this unused `db` import along with the `monkeypatch.setattr(worker, 'db', db)` line, and the unit tests work fine again. I think we should leave it as is for now. It's been in the code for 3+ years to no ill effect, and we want to limit the number of changes we merge back into main.